src/assets/interoperability.ts

```diff
@@ -11,48 +11,18 @@
  *
  * Removal or modification of this copyright notice is prohibited.
  */
-import {
-	MODULE_NAME_INTEROPERABILITY,
-	CHAIN_ID_LENGTH,
-	CHAIN_NAME_MAINCHAIN,
-	EMPTY_BYTES,
```

Comment:
Please check and remove any unnecessary constants from the file, if they're not being used elsewhere.